src/components/flow/routers/ticket/helpers.ts

```diff
@@ -28,10 +28,14 @@ export const nodeToState = (settings: NodeEditorSettings): TicketRouterFormState
   const router = settings.originalNode.node.router as SwitchRouter;
 
   let note = { value: '' };
-  let resultName = { value: router.result_name || '' };
+  let resultName = { value: '' };
   let assignee: FormEntry = { value: null };
   let topic: FormEntry = { value: null };
 
+  if (router) {
+    resultName = { value: router.result_name };
```

Comment:
Now I think we need `resultName = { value: router.result_name || '' };` just in case the route has undefined result_name